### PR TITLE
implementing destroy method

### PR DESCRIPTION
**Pull request by nehamamandelbaum**
This PR solves issue #570, which was to implement the delete method to the professional experiences in the user interface. 

How the UI looks: 

Captura de Tela 2023-10-30 às 14 51 15
Captura de Tela 2023-10-30 às 14 51 28
Captura de Tela 2023-10-30 às 14 51 36

### CODE REVIEW

File: app/controllers/professional_experiences_controller.rb [hunk 1]
```diff
@@ -51,6 +51,16 @@ def update
     redirect_to '/404'
   end
 
+  def destroy
+    @professional_experience = scoped_professional_experiences.find(params[:id])
+    @professional_experience.destroy
+    redirect_to professional_experiences_path,
+                notice: I18n.t(:notice, scope: "flash.actions.destroy",
+                                        resource_name: ProfessionalExperience.model_name.human)
+  rescue ActiveRecord::RecordNotFound
```
Comment (adbatista) [inline app/controllers/professional_experiences_controller.rb:60]: da para usar o rescue_from ao invés de  fazer isso em toda action https://api.rubyonrails.org/classes/ActiveSupport/Rescuable/ClassMethods.html#method-i-rescue_from

File: app/controllers/professional_experiences_controller.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class ProfessionalExperiencesController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound do |_|
```
Comment (adbatista) [inline app/controllers/professional_experiences_controller.rb:2]: if there's no use for the argument, you don't need to add here

` rescue_from ActiveRecord::RecordNotFound do` should be fine

**Comment by adbatista** [comment]
conflicts

**Comment by nehamamandelbaum** [comment]
> conflicts

Resolved!

**Pull request merged**